lib/datadog/tracing/contrib/active_support/cache/instrumentation.rb

```diff
@@ -10,96 +10,63 @@ module Contrib
       module ActiveSupport
         module Cache
           # Defines instrumentation for ActiveSupport caching
-          # rubocop:disable Lint/RescueException
           module Instrumentation
             module_function
 
-            def start_trace_cache(payload)
```

Comment:
Look at the new code separately from old code is probably the easiest way to review this file.